### ISSUE TITLE
A couple of Editor thumbnails are still using old theme

### ISSUE THREAD

**Issue by JohnSmith-LT** [thexerteproject/xerteonlinetoolkits#1106]
![image](https://user-images.githubusercontent.com/3907362/171733824-45093492-1859-468b-ab08-b4abf549fa2a.png)

![image](https://user-images.githubusercontent.com/3907362/171734131-8b29da5c-4451-48a4-a458-e3b1cdd904aa.png)

**Comment by ronm123** [comment]
@JohnSmith-LT I've created an example of the Timeline Media page in our page types demonstrator https://xot.xerte.org.uk/play_116#timelinemedia so have added and committed an updated thumbnail for that page type.
I don't have a good example of the interactive video page type so haven't created an example or new thumb for that.

Going through that thumbnail folder brings back distant memories! ;-)

**Comment by FayCross** [comment]
@JohnSmith-LT no, I'm afraid not. @ingedonke - do you have a good example of the interactive video page that we could use in the page examples project & screenshot for the insert menu?

**Comment by ingedonke** [comment]
Hi Fay,

This is the link to the Dutch examples (I shared this project with you 
in xot.xerte.urg.uk). Maybe this is something to use?
https://xot.xerte.org.uk/preview_164#interactievevideo

Bye,
Inge

<https://github.com/JohnSmith-LT>
Logo_DLearning

Inge Donkervoort
Tuinstraat 8c
5241 AA Rosmalen

*T* +31 (0)85-1308364
*M* +31 6-50267104
*E* ***@***.***
*I* www.dlearning.nl <http://www.dlearning.nl>

*F* facebook <https://www.facebook.com/dlearning1>
*T* twitter <https://twitter.com/DlearningInge>
*L* linkedIn <http://nl.linkedin.com/pub/inge-donkervoort/7/4b/18a>

/Dit e-mailbericht is slechts bestemd voor de persoon of organisatie aan 
wie het is gericht. Het bericht kan informatie bevatten die persoonlijk 
of vertrouwelijk is. Tevens kan het informatie bevatten die niet 
publiekstoegankelijk van aard is. DLearning aanvaardt geen 
aansprakelijkheid indien de inhoud van dit bericht en eventuele bijlagen 
in strijd is met de wet of de goede zeden. Voor het geval de ontvanger 
van dit bericht niet de bedoelde persoon of organisatie is, wordt 
hierbij vermeld dat verdere verspreiding, openbaarmaking of 
vermenigvuldiging van dit bericht strikt verboden is en verzoeken wij u 
de inhoud niet te gebruiken en de afzender direct te informeren. Dank 
voor uw medewerking./

Op 8-6-2022 om 11:37 schreef FayCross:

**Comment by ronm123** [comment]
@ingedonke @FayCross @JohnSmith-LT I've drafted a possible example of the interactive video page to use in our page types demo and then obviously to generate the thumbnail too. However in doing so I'm noticing some issues with this page type that means it doesn't really work properly on smartphones - at least not my iPhone. Got a meeting shotly but will add a separate issue and list what I've found but for instance if you view this demo on your smart phone https://xot.xerte.org.uk/play_335#page1 what I get is inconsistent layering of the overlays e.g the MC questions seem fine but some of the overlays appear below rather than above the video - does that happen for you too? On desktop and iPad all seems fine.
As you'll see I've also included a YouTube and Vimeo version. I would prefer to use the YouTube version but I don't think there's a way now to remove the 'more videos' panel which you used to be able to do by adding rel=0 to the url. I know this probably doesn't happen with PeerTube but that's not a realistic solution for most of our users.

Anyway if you think this is a suitable example I'll import it into the English page types demonstrator?

**Comment by ingedonke** [comment]
Hi,
I think this is a suitable (good) example. The layers are fine on my Phone only the categories page is a bit difficult to use on the Phone. But thats ok.

Bye,
Inge

Bye,
Inge

Ron Mitchell ***@***.***> schreef op 9 juni 2022 10:44:41 CEST:
***@***.*** @FayCross @JohnSmith-LT I've drafted a possible example of the interactive video page to use in our page types demo and then obviously to generate the thumbnail too. However in doing so I'm noticing some issues with this page type that means it doesn't really work properly on smartphones - at least not my iPhone. Got a meeting shotly but will add a separate issue and list what I've found but for instance if you view this demo on your smart phone https://xot.xerte.org.uk/play_335#page1 what I get is inconsistent layering of the overlays e.g the MC questions seem fine but some of the overlays appear below rather than above the video - does that happen for you too? On desktop and iPad all seems fine.
>As you'll see I've also included a YouTube and Vimeo version. I would prefer to use the YouTube version but I don't think there's a way now to remove the 'more videos' panel which you used to be able to do by adding rel=0 to the url. I know this probably doesn't happen with PeerTube but that's not a realistic solution for most of our users.
>
>Anyway if you think this is a suitable example I'll import it into the English page types demonstrator?
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/thexerteproject/xerteonlinetoolkits/issues/1106#issuecomment-1150843962
>You are receiving this because you were mentioned.
>
>Message ID: ***@***.***>
-- 
Verstuurd vanaf mijn Android apparaat met K-9 Mail. Excuseer mijn beknoptheid.

**Comment by JohnSmith-LT** [comment]
Not sure about a real iPhone (or any other device for that matter) but on the iPhone simulator in Chrome it's often really difficult to see where the Play button actually is in order to be able to click this to move on... Is there a better way that this could be achieved or is it a non issue in real life? Seems like a close button or a Play button actually on the overlay would be better as it would also allow the overlay to be full with/height if needed on phones with small screens...

![image](https://user-images.githubusercontent.com/3907362/172807887-98182e89-bcd4-4aa4-8dcb-a2cd8b39c838.png)

**Comment by JohnSmith-LT** [comment]
Also, not sure if anyone can reproduce this bug, but if you go to page 2 and then return to page 1 (of your LO @ronm123) then the video player can get stuck in some kind of strange play/pause condition where both buttons are showing in the two different control bars at the same time but you cannot click either so the video is now stuck:

![image](https://user-images.githubusercontent.com/3907362/172810440-6dc721e9-4bc3-4e13-8b8a-5d52bc60f26e.png)

**Comment by JohnSmith-LT** [comment]
Hi @ronm123 I've created a separate issue with some of my findings although I'm not seeing the issue of overlays (underlays) not appearing layered correctly... I am using the Chrome iPhone simulator though and not a real device so this could be a Safari thing...

**Comment by ronm123** [comment]
@JohnSmith-LT have added the example to the page types demo now https://xot.xerte.org.uk/play_116# and committed the thumbnail too. I think we're missing some other thumbnails/examples so won't close this issue yet.

**Comment by ronm123** [comment]
@JohnSmith-LT @FayCross @torinfo @ingedonke while I was working on the English page types demo to add the thumbnails and examples of timeline media and interactive video as reported by John in this issue I noticed that we only have place holder pages for 2 other page types and also a strange issue with the Wordsearch page. I think the missing example pages may be listed/discussed elsewhere but may as well make note of them here too:

Image sequence - we have a suitable thumbnail but not a working example and I guess the current thumbnail may need to change if the example is different but I think @FayCross has this as a todo at some point?

Highlight text - we seem to have a suitable thumbnail but not a working example?

Wordsearch - we have the thumbnail and example but I noticed that in our online copy https://xot.xerte.org.uk/play_116#wordsearch the instruction is below the wordsearch panel whereas in my local develop copy the instruction is on the left and the panel on the right. I guess this might just be a difference between develop and 3.10.4 and will be fixed when the install is updated?
